subprojects/nanoarrow.wrap

```diff
@@ -0,0 +1,9 @@
+[wrap-file]
```

Comment:
Ideally we would use the meson WrapDB entry, but in this case there's an upstream bug in nanoarrow that prevents us from using the latest release (0.6.0). See also https://github.com/apache/arrow-nanoarrow/pull/702

Whenever 0.7.0 gets released we can go back to using the Meson WrapDB entry, although I don't think there is any huge rush for that either